### PR TITLE
[#10] Use non modified binaries

### PR DESCRIPTION
**Pull request by rvem**
https://gitlab.com/tezos/tezos/-/merge_requests/1624 was merged, we can use non-modified static
binaries from https://github.com/serokell/tezos-packaging now.

Update scripts and instructions to use these binaries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,29 +49,25 @@ docker volume create ubuntu-tezos-volume
...
 ### Without docker
 
 First step for running the private blockchain is generating a new genesis public key and
-building patched Tezos binaries. [`build-patched-binaries.sh`](./scripts/build-patched-binaries.sh)
-shell script will build these patched binaries.
-
-Note that you may have to adjust this script or [`patch_template.patch`](./patches/patch_template.patch)
-for your needs, e.g. genesis public keys can be moved from
-`src/proto_genesis_{babylonnet, carthagenet}/lib_protocol/data.ml` to another files.
+fetching Tezos binaries. [`fetch-binaries.sh`](./scripts/fetch-binaries.sh)
```
Comment (marklnichols) [inline README.md:61]: Just changing the wording a bit:

fetching Tezos binaries. The [`fetch-binaries.sh`](./scripts/fetch-binaries.sh)
shell script will build these patched binaries.
This shell script will fetch statically built binaries and create a tezos-node config file
containing a custom genesis key.  This custom genesis key will be used by the script `start-baker.sh`.

File: README.md [hunk 2]
```diff
@@ -176,7 +149,7 @@ To run baker inside docker container run the following:
 # Note that here you should specify the port using which your node can be
 # accessed, thus you also need to expose and publish this port for docker.
 docker run --expose 8733 -p 8733:8733 -v ubuntu-tezos-volume:/base-dir \
-  -i -t ubuntu-tezos start-baker --net-addr-port 8733 --base-chain carthagenet \
+  -i -t ubuntu-tezos start-baker --net-addr-port 8733 \
```
Comment (marklnichols) [inline README.md:152]: If we do want to use Docker outside of this tutorial, we are going to need the full net-addr, not just the port.  The Container IP only works for the multiple-machines-locally tutorial.

File: scripts/start-baker.sh [hunk 3]
```diff
@@ -13,19 +13,7 @@ gen_node_identity() {
...
-        carthagenet )
-            network="carthagenet"
-            ;;
-        *)
-            echo "$base_chain not supported. Only 'babylonnet' and 'carthagenet' are supported."
-            exit 1
-            ;;
-    esac
-    node_args=("--data-dir" "$node_dir" "--rpc-addr" "localhost:8732" "--net-addr" "$net_addr" "--no-bootstrap-peers" "--bootstrap-threshold" "1" "--network" "$network")
+    node_args=("--data-dir" "$node_dir" "--rpc-addr" "localhost:8732" "--net-addr" "$net_addr" "--no-bootstrap-peers" "--bootstrap-threshold" "1")
```
Comment (marklnichols) [inline scripts/start-baker.sh:16]: Do we not need $network now?

**Review by marklnichols — APPROVED** [review]
Just some minor comments

File: scripts/start-baker.sh (hunk 3 above)
Comment (rvem) [reply scripts/start-baker.sh:16]: no, we provide network information in config file now

File: README.md (hunk 2 above)
Comment (rvem) [reply README.md:152]: Let's move this to a separate issue

Comment (rvem) [reply README.md:152]: And deal with it once your PR is merged as well

File: README.md [hunk 4]
```diff
@@ -143,8 +117,7 @@ Here is an example of script usage:
 ./scripts/start-baker.sh --base-dir baker \
   --tezos-client baker/tezos-client --tezos-node baker/tezos-node \
   --tezos-baker baker/tezos-baker-005-PsBabyM1 --tezos-endorser baker/tezos-endorser-005-PsBabyM1 \
-  --net-addr 10.147.19.192:8732
-  --peer 10.147.19.49:8732 --base-chain carthagenet
+  --net-addr 10.147.19.192:8732 --peer 10.147.19.49:8732
```
Comment (marklnichols) [inline README.md:120]: Should these be the ...006-PsCARTHA variations?

Comment (rvem) [reply README.md:120]: hm, you're right

**Comment by gromakovsky** [comment]
Doesn't it close #10? Why no PR template?

**Comment by rvem** [comment]
Forgot about it

**Pull request merged**